### ISSUE TITLE
Adding the check for detected a diferent file format

### ISSUE THREAD

**Issue by JesusZapata** [PyCQA/pylint#1616]
## Steps to reproduce
I think of pylint can be detected the newline using the `pylintrc` file

Each OS's has newline diferent
Windows `\r\n`
Linux `\n`
MAC `\r` 

In favor of all code has the same newline character

My idea is:
The file `pylintrc`
```
file_format=unix
```
Then pylint detected another newline diferent to the unix 

### Current behavior
Pylint not detected the newline

### Expected behavior
Pylint detect the newline using the file of configuration file `pylintrc` using the configuration `file_format=unix`

### pylint --version output
```
pylint 2.0.0, 
astroid 1.5.0
Python 2.7.12 (default, Jul 18 2016, 15:02:52) 
[GCC 4.8.4]
```
Is only one idea

**Comment by PCManticore** [comment]
I find this to be too nitpicky for pylint.

**Comment by moylop260** [comment]
FYI It looks similar to eslint check [linebreak-style](http://eslint.org/docs/rules/linebreak-style)

**Comment by rogalski** [comment]
Aren't we are having this one already in form of [`expected-line-ending-format`](https://github.com/PyCQA/pylint/blob/126c63db8010d23afd18cf65e4a6b45f4bf79efd/pylint/checkers/format.py#L479) ?
